tests/test_utils.py

```diff
@@ -211,6 +211,20 @@ def test_tracks_to_napari_incorrect_ndim(ndim: int):
         data, properties, graph = utils.tracks_to_napari(tracks, ndim=ndim)
 
 
+@pytest.mark.parametrize("ndim", [2, 3])
+def test_tracks_to_napari_ndim_inference(ndim: int):
+    """Test inferring the correct dimensions from track data when using
+    `tracks_to_napari`."""
+
+    # make three fake tracks with properties
```

Comment:
is this comment correct? It looks like only 1 fake track is being created

Reply:
I think that's fine for this test, since it only needs one to test the dimensionality

Reply:
ah yep, a single track is fine but I think the comment:

```
# make three fake tracks with properties
```

should be changed to:

```
# make a fake track with properties
```

just to make it clear for future people looking at the test